src/store/projects.js

```diff
@@ -0,0 +1,38 @@
+import { writable } from 'svelte/store';
+import orderOptions from '@/constants/projects/order';
+import competencesOptions from '@/constants/projects/competences';
+
+export const filters = writable({
+  competences: competencesOptions,
+  spots: 1,
+  date: {},
+  search: '',
+});
+
+export const selectedOrder = writable(orderOptions[0]);
+
+export function selectOrder(value) {
+  selectedOrder.update(() => value);
+}
+
+function changeFilter(name, value) {
+  filters.update((state) => ({...state, [name]: value}));
+}
+
+export function changeSearch(value) {
+  changeFilter('search', value);
+}
+
+export function changeSpots(value) {
+  if (value > 0) changeFilter('spots', value);
```

Comment:
A simpler way would be to write 
```suggestion
  if (value > 0) value = 1;
```
Or even 
```suggestion
  value = Math.max(1, value);
```
and just use the variable `value` right away

Reply:
Oh maaan, you're a math profi

Reply:
I just don't like to repeat myself as much as possible 😁